src/ActionButton/__stories__/actionbuttton.stories.js

```diff
@@ -22,18 +23,105 @@ export default {
   title: 'components/ActionButton'
 }
 
-export const actionbuttons = () => {
+export const ActionButtonDefault = () => {
   const props = {
-    label: 'stable',
-    onClick: () => {
+    onPress: () => {
       console.log('button was clicked')
-    },
-    icon: <Bug size='S' />
+    }
   }
 
   return (
     <Provider theme={theme} colorScheme='light'>
```

Comment:
Since every story uses the same `Provider` wrapper setup, we can probably extract it into a decorator for the stories in this file:

```jsx
const Wrapper = storyFn => {
  return <Provider theme={theme} colorScheme='light'>{storyFn()}</Provider>
}
export default {
  title: 'components/ActionButton',
  decorators: [Wrapper]
}
```

See: https://storybook.js.org/docs/addons/introduction/#1-decorators

Reply:
Great idea, I've made that change and will do it for future stories.